wallets/provider-all/src/helpers.ts

```diff
@@ -0,0 +1,12 @@
+import type { ProviderInterface } from '@rango-dev/wallets-react';
+import type { WalletType } from '@rango-dev/wallets-shared';
+
+import { WalletTypes } from '@rango-dev/wallets-shared';
+
+export const isWalletConnectExcluded = (
+  selectedProviders?: (WalletType | ProviderInterface)[]
+) =>
+  selectedProviders &&
+  !selectedProviders.find(
```

Comment:
You are only checking string mode of the union (`selectedProviders`).

I guess it should be something like this:
```ts
  !selectedProviders.find((provider) => {
    if (typeof provider === 'string') {
      return provider === WalletTypes.WALLET_CONNECT_2;
    }
    return provider.getWalletInfo([]).name;
  });
```

Reply:
Tnx, Fixed